### PR TITLE
Update setup steps in README

### PR DESCRIPTION
**Pull request by bluecmd**
This should reflect what I needed to do on a fresh Debian installation today.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,9 +31,11 @@ Build Requirements
 Building oreboot
 -----------------
 
+To setup your Rust environment for oreboot, see below:
+
 ```
-# set up the environment
-source ENV
+# Install some tools we will need later
+sudo apt install device-tree-compiler pkg-config
```
Comment (rminnich) [inline README.md]: why did you remove 
source ENV
? I use it.

Comment (bluecmd) [reply README.md]: Ah, I thought it was an old file - it seems it's a new file (my branch didn't have it).
I'll re-add it. I didn't have the targets/ directory either.

**Comment by bluecmd** [comment]
PTAL

File: README.md [hunk 2]
```diff
@@ -45,15 +47,22 @@ rustup override set nightly
...
-
-# Install a few compiler tools.
+# Install a few compiler tools
+# NOTE: This needs to be run in the root of the oreboot directory
+cd oreboot/
 cargo make setup
-sudo apt-get install device-tree-compiler
+```
 
+Occationally you want to run `rustup update` to keep your Rust up-to-date.
```
Comment (rminnich) [inline README.md]: occasionally

File: README.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ Oreboot+QEMU for RISC-V HiFive Unleased:
 
 [![asciicast](https://asciinema.org/a/XnWkMWTABuajsbGPMMTefjuZ2.svg)](https://asciinema.org/a/XnWkMWTABuajsbGPMMTefjuZ2)
 
-Oreboot+QEMU for AST2500:
+Oreboot+QEMU for ARM:
```
Comment (rminnich) [inline README.md]: I think you need a note in here somewhere about export OREBOOT into the environment. That made x86 builds work anyway.

File: README.md [hunk 4]
```diff
@@ -45,15 +47,22 @@ rustup override set nightly
...
+Occasionally you want to run `rustup update` to keep your Rust up-to-date.
+A good idea is to make sure you have the latest version before reporting any
+issues.
+
+To build oreboot for a specific platform, do like this:
+
+```
 # Build for RISC-V
-cd src/mainboard/sifive/hifive
+source oreboot/ENV
```
Comment (rminnich) [inline README.md]: that file is gone, right?

**Comment by bluecmd** [comment]
PTAL. This is probably the world's best README.md as of now.
